microsoft/testsuites/display/drm.py

```diff
@@ -73,6 +73,12 @@ def verify_drm_driver(self, node: Node, log: Logger) -> None:
         ),
     )
     def verify_dri_node(self, node: Node, log: Logger) -> None:
+        lsmod = node.tools[Lsmod]
+        if lsmod.module_exists("hyperv_fb"):
+            # https://dev.azure.com/msazure/LSG-linux/_wiki/wikis/LSG%20ICM%20Document/650798/Display
+            raise SkippedException(
+                "hyperv_fb module is present, so dri node is not populated"
+            )
```

Comment:
- Please check whether we can unload the fb module to continue the following testing.
- Move it into before_case if we need skip under this situation.

Reply:
Instead of unloading, I am blacklisting in grub cmdline.
Moved it to before_case.